JavaCryptographicArchitecture/src/SSLEngine.crysl

```diff
@@ -1,38 +1,14 @@
 SPEC javax.net.ssl.SSLEngine
 
-/*
- *  When using raw SSLSocket and SSLEngine classes, 
- *  you should always check the peer's credentials
- *  before sending any data. The SSLSocket and SSLEngine 
```

Comment:
Why are you removing almost everything from this rule?

Reply:
1. the constructors of the rule are protected
2. the aggregates aren't required
3. the comments have no primary value for the rule

Reply:
The object will be created by the javax.net.ssl.SSLContext class